test/module/irohad/network/block_loader_test.cpp

```diff
@@ -250,24 +253,20 @@ TEST_F(BlockLoaderTest, ValidWhenMultipleBlocks) {
  */
 TEST_F(BlockLoaderTest, ValidWhenBlockPresent) {
   // Request existing block => success
-  auto requested = wrapBlock(std::make_shared<shared_model::proto::Block>(
-      getBaseBlockBuilder().build().signAndAddSignature(key).finish()));
+  auto block = std::make_shared<shared_model::proto::Block>(
+      getBaseBlockBuilder().build().signAndAddSignature(key).finish());
+  auto requested = wrapBlock(block);
   block_cache->insert(requested);
 
   EXPECT_CALL(*peer_query, getLedgerPeers())
       .WillOnce(Return(std::vector<wPeer>{peer}));
-  EXPECT_CALL(*validator, validate(*requested)).WillOnce(Return(Answer{}));
+  EXPECT_CALL(*validator, validate(RefAndPointerEq(block)))
```

Comment:
Is it possible to write `validate(*block)` instead, so that `RefAndPointerEq` is not required?

Reply:
Then it tries to copy abstract block